cmd/devinit/utils.go

```diff
@@ -218,10 +218,16 @@ func getOrCreateRandomSecret(length int, filePaths ...string) ([]byte, error) {
 	}
 
 	if exists {
-		return os.ReadFile(filePath)
+		fileContent, err := os.ReadFile(filePath)
+		if err != nil {
+			return nil, err
+		}
+		if len(fileContent) != 0 {
+			return fileContent, nil
+		}
```

Comment:
Should this return something if `len(fileContent) == 0`? Previously it would return here, but now it will continue.